### PR TITLE
implement timming of applicant section in csp form

### PR DESCRIPTION
**Pull request by tfang54321**

### CODE REVIEW

File: apps/csp/src/app/form-base/form-base.component.ts [hunk 1]
```diff
@@ -215,10 +219,10 @@ export class FormBaseComponent implements OnInit, AfterViewInit {
     this.processErrors();
   }
 
-  // processRegulatoryInfoErrors(errorList) {
-  //   this._regulatoryInfoErrors = errorList;
-  //   this.processErrors();
-  // }
+  processTimmingOfApplicantErrors(errorList) {
```
Comment (AMckayHC) [inline apps/csp/src/app/form-base/form-base.component.ts]: processTimming - can you remove the extra m in processTiming

File: apps/csp/src/app/global/global.service.ts [hunk 2]
```diff
@@ -28,7 +28,9 @@ export class GlobalService {
   private _stateList: ICode[];
   private _yesnoList: ICode[];
   private _dossierTypes: ICodeDefinition[] = [];
-  private _drugUses: ICode[] = [];;
+  private _drugUses: ICode[] = [];
+  private _timmingOfApplicant: ICodeAria[] = [];
```
Comment (AMckayHC) [inline apps/csp/src/app/global/global.service.ts]: remove extra m - _timming

**Review by AMckayHC — CHANGES_REQUESTED** [review]

File: apps/csp/src/app/global/global.service.ts (hunk 2 above)
Comment (tfang54321) [reply apps/csp/src/app/global/global.service.ts]: fixed

File: apps/csp/src/app/form-base/form-base.component.ts (hunk 1 above)
Comment (tfang54321) [reply apps/csp/src/app/form-base/form-base.component.ts]: fixed

**Review by AMckayHC — APPROVED** [review]

**Pull request merged**